study.md

```diff
@@ -160,15 +163,17 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+A response comes from the server and responds to the client request.
+
+My personal favorite response code is 301 - when a url has moved.
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+I'm incredbily confused on what curl actually is or what it does or how to use it.
```

Comment:
Curl is a tool that we can use to make a request to a server without having a typical client. If you wait until you have a front-end that makes AJAX requests, it will be more difficult to debug if something goes wrong.

If you can successfully make a curl request to an API and get a success, you at least know how to interact with the API, how to structure the data, and that the API works how you think it does.

Hope this helps! I was very confused about curl requests at first too, its really just a way to validate what you think you know about how the server expects to receive data, before you try to incorporate it into your actual front-end.